### ISSUE TITLE
Multi-page longtables generate multiple entries in listoftables.

### ISSUE THREAD

**Issue by MichaelHatherly** [ronisbr/PrettyTables.jl#95]
(Hopefully I've not overlooked some configuration for this somewhere...)

Since there is no `\endfirsthead` (there is only a `\endhead`) in `longtable` output the `\caption` gets captured in the `\listoftables` for however many pages the table spans. Headers on subsequent pages should be using `\caption*` to avoid them getting included in the LoT.

Basically we need something like:

```latex
\begin{longtable}{rrrr}
  \caption{A caption for this table.}\\
  \hline\hline
  \textbf{a} & \textbf{b} & \textbf{c} & \textbf{d} \\
  \texttt{Int64} & \texttt{Int64} & \texttt{Int64} & \texttt{Int64} \\\hline
  \endfirsthead
  \caption*{A caption for this table.}\\ %%% caption* used here
  \hline\hline
  \textbf{a} & \textbf{b} & \textbf{c} & \textbf{d} \\
  \texttt{Int64} & \texttt{Int64} & \texttt{Int64} & \texttt{Int64} \\\hline
  \endhead
  \hline\hline
  \endfoot
  \endlastfoot
```

instead of

```latex
\begin{longtable}{rrrr}
  \caption{A caption for this table.}\\
  \hline\hline
  \textbf{a} & \textbf{b} & \textbf{c} & \textbf{d} \\
  \texttt{Int64} & \texttt{Int64} & \texttt{Int64} & \texttt{Int64} \\\hline
  \endhead
  \hline\hline
  \endfoot
  \endlastfoot
```

for the resulting LoT to not show duplicates.

Great package btw, thanks!

**Comment by ronisbr** [comment]
Nice! Thanks for the suggestion, I will modify this. It seems very easy to do.

> Great package btw, thanks!

Thanks! I am glad it is being useful :)

**Comment by ronisbr** [comment]
What about:

```latex
\begin{longtable}{rrrr}
  \caption{A caption for this table.}\\
  \hline\hline
  \textbf{a} & \textbf{b} & \textbf{c} & \textbf{d} \\
  \texttt{Int64} & \texttt{Int64} & \texttt{Int64} & \texttt{Int64} \\\hline
  \endfirsthead
  \hline\hline
  \textbf{a} & \textbf{b} & \textbf{c} & \textbf{d} \\
  \texttt{Int64} & \texttt{Int64} & \texttt{Int64} & \texttt{Int64} \\\hline
  \endhead
  \hline\hline
  \endfoot
  \endlastfoot
```

Do we really need a caption at every page with `longtable`?

**Comment by MichaelHatherly** [comment]
> Do we really need a caption at every page with `longtable`?

Agreed, probably doesn't need one per page.

**Comment by ronisbr** [comment]
@MichaelHatherly done! Can you please test using `master`?

**Comment by MichaelHatherly** [comment]
Yes, that works for me. Thanks. Would you be willing to tag a new release with this?

**Comment by ronisbr** [comment]
Yes! I will do this shortly (perhaps today).

**Comment by MichaelHatherly** [comment]
Thanks. Would it also include `wrap_table` (#85) which appears in `0.10.1` release notes, but doesn't seem to actually be included from what I've tried?

**Comment by ronisbr** [comment]
`wrap_table` should be working in 0.10.1. Can you tell me what is going on? Maybe you found another bug :)

**Comment by MichaelHatherly** [comment]
```
(@v1.5) pkg> st PrettyTables
Status `~/.julia/environments/v1.5/Project.toml`
  [08abe8d2] PrettyTables v0.10.1

julia> using PrettyTables, DataFrames

julia> pretty_table(DataFrame(a = 1:2); backend=:latex, wrap_table=false)
ERROR: MethodError: no method matching _pt_latex(::IOContext{Base.TTY}, ::PrettyTables.PrintInfo; wrap_table=false)
Closest candidates are:
  _pt_latex(::IO, ::PrettyTables.PrintInfo; tf, body_hlines, cell_alignment, highlighters, hlines, longtable_footer, noheader, nosubheader, row_number_alignment, table_type, vlines) at /home/mike/.julia/packages/PrettyTables/W16qB/src/backends/latex/print.jl:10 got unsupported keyword argument "wrap_table"
Stacktrace:
 [1] kwerr(::NamedTuple{(:wrap_table,),Tuple{Bool}}, ::Function, ::IOContext{Base.TTY}, ::PrettyTables.PrintInfo) at ./error.jl:157
 [2] _pt(::IOContext{Base.TTY}, ::PrettyTables.ColumnTable, ::Array{Any,1}; alignment::Symbol, backend::Symbol, cell_alignment::Nothing, cell_first_line_only::Bool, compact_printing::Bool, filters_row::Nothing, filters_col::Nothing, formatters::Nothing, header_alignment::Symbol, header_cell_alignment::Nothing, renderer::Symbol, row_names::Nothing, row_name_alignment::Symbol, row_name_column_title::String, row_number_column_title::String, show_row_number::Bool, title::String, title_alignment::Symbol, kwargs::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:wrap_table,),Tuple{Bool}}}) at /home/mike/.julia/packages/PrettyTables/W16qB/src/private.jl:426
 [3] _pretty_table(::IOContext{Base.TTY}, ::DataFrame, ::Array{String,1}; kwargs::Base.Iterators.Pairs{Symbol,Any,Tuple{Symbol,Symbol},NamedTuple{(:backend, :wrap_table),Tuple{Symbol,Bool}}}) at /home/mike/.julia/packages/PrettyTables/W16qB/src/private.jl:356
 [4] #pretty_table#46 at /home/mike/.julia/packages/PrettyTables/W16qB/src/print.jl:670 [inlined]
 [5] top-level scope at REPL[2]:1
```

**Comment by MichaelHatherly** [comment]
With current `master` it does work:

```
(@v1.5) pkg> st PrettyTables
Status `~/.julia/environments/v1.5/Project.toml`
  [08abe8d2] PrettyTables v0.10.0 `~/.julia/dev/PrettyTables`

julia> using PrettyTables, DataFrames
[ Info: Precompiling DataFrames [a93c6f00-e57d-5684-b7b6-d8193f3e46c0]

julia> pretty_table(DataFrame(a = 1:2); backend=:latex, wrap_table=false)
\begin{tabular}{r}
  \hline\hline
  \textbf{a} \\
  \texttt{Int64} \\\hline
  1 \\
  2 \\\hline\hline
\end{tabular}

julia> pretty_table(DataFrame(a = 1:2); backend=:latex)
\begin{table}
  \begin{tabular}{r}
    \hline\hline
    \textbf{a} \\
    \texttt{Int64} \\\hline
    1 \\
    2 \\\hline\hline
  \end{tabular}
\end{table}
```

**Comment by ronisbr** [comment]
Oh sorry! My mistake! I will be available in v0.11. I will fix CHANGELOG.md.

**Comment by MichaelHatherly** [comment]
Might be TagBot issue then, since https://github.com/ronisbr/PrettyTables.jl/releases/tag/v0.10.1 lists #85. No worries though, if it's all available in forthcoming `0.11` that's great.

**Comment by ronisbr** [comment]
Yeah, TagBot list everything that was closed. This issue was closed, but I was already working on `v0.11`, which should have more changes. Then, I create a branch to release 0.10.1 with a bug fix.

**Comment by ronisbr** [comment]
Thanks for pointing this out! I need to take more care next time with Tagbot :)
